client/containers/Pub/PubDocument/PubBottom/Discussions/LabelFilter.tsx

```diff
@@ -205,8 +205,8 @@ class LabelFilter extends Component<Props, State> {
 												content={
 													label.publicApply ? (
 														<span>
-															All discussion authors can apply this
-															label.
+															All discussion contributors can apply
```

Comment:
Again, strikes me as a bit awkward here, since we we're now making Contributors an important word, and this isn't really an instance of it (at least not now). So, perhaps: `Anyone can apply this label to their discussions.`

Reply:
Yes, this is a great point. Some discussants may qualify as recognized contributors, but most won't. "Anyone" works better than "contributor," and I think that more than suffices, but I also think it's useful to specify for managers that only logged-in/authenticated users--`members` seems the best word here--will be able to discuss and use these labels. So, `All members can apply this label.`